### ISSUE TITLE
Unit Tests keep reporting 404 for imports

### ISSUE THREAD

**Issue by johnlim** [PolymerLabs/todo-list#14]
Hi,

I'm trying to add unit test for the todo-view but keep getting 404 errors on the console like so 

```
Installing and starting Selenium server for local browsers
Selenium server running on port 59182
Web server running on port 2000 and serving from /Users/git/todo-list
chrome 48                Beginning tests via http://localhost:2000/components/todo-list/generated-index.html?cli_browser_id=0
chrome 51                Beginning tests via http://localhost:2000/components/todo-list/generated-index.html?cli_browser_id=1
firefox 40               Beginning tests via http://localhost:2000/components/todo-list/generated-index.html?cli_browser_id=2
404 GET /components/todo-list/app/bower_components/iron-flex-layout/classes/iron-flex-layout.html
404 GET /components/todo-list/app/bower_components/paper-drawer-panel/paper-drawer-panel.html
404 GET /components/todo-list/app/bower_components/paper-header-panel/paper-header-panel.html
404 GET /components/todo-list/app/bower_components/paper-icon-button/paper-icon-button.html
404 GET /components/todo-list/app/bower_components/paper-menu/paper-menu.html
404 GET /components/todo-list/app/bower_components/paper-toolbar/paper-toolbar.html
404 GET /components/todo-list/app/bower_components/iron-iconset-svg/iron-iconset-svg.html
404 GET /components/todo-list/app/bower_components/polymer/polymer.html
404 GET /components/todo-list/app/bower_components/paper-button/paper-button.html
404 GET /components/todo-list/app/bower_components/paper-input/paper-input.html
404 GET /components/todo-list/app/bower_components/paper-checkbox/paper-checkbox.html
404 GET /components/todo-list/app/bower_components/paper-material/paper-material.html
404 GET /components/todo-list/app/bower_components/iron-flex-layout/classes/iron-flex-layout.html
404 GET /components/todo-list/app/bower_components/paper-drawer-panel/paper-drawer-panel.html
404 GET /components/todo-list/app/bower_components/paper-header-panel/paper-header-panel.html
404 GET /components/todo-list/app/bower_components/paper-icon-button/paper-icon-button.html
404 GET /components/todo-list/app/bower_components/paper-menu/paper-menu.html
404 GET /components/todo-list/app/bower_components/paper-toolbar/paper-toolbar.html
404 GET /components/todo-list/app/bower_components/polymer/polymer.html
404 GET /components/todo-list/app/bower_components/paper-button/paper-button.html
404 GET /components/todo-list/app/bower_components/paper-input/paper-input.html
404 GET /components/todo-list/app/bower_components/iron-iconset-svg/iron-iconset-svg.html
404 GET /components/todo-list/app/bower_components/paper-checkbox/paper-checkbox.html
404 GET /components/todo-list/app/bower_components/paper-material/paper-material.html
chrome 51                Tests passed
chrome 48                Tests passed
404 GET /components/todo-list/app/bower_components/iron-flex-layout/classes/iron-flex-layout.html
404 GET /components/todo-list/app/bower_components/paper-drawer-panel/paper-drawer-panel.html
404 GET /components/todo-list/app/bower_components/paper-header-panel/paper-header-panel.html
404 GET /components/todo-list/app/bower_components/paper-icon-button/paper-icon-button.html
404 GET /components/todo-list/app/bower_components/paper-menu/paper-menu.html
404 GET /components/todo-list/app/bower_components/paper-toolbar/paper-toolbar.html
404 GET /components/todo-list/app/bower_components/polymer/polymer.html
404 GET /components/todo-list/app/bower_components/paper-button/paper-button.html
404 GET /components/todo-list/app/bower_components/paper-input/paper-input.html
404 GET /components/todo-list/app/bower_components/iron-iconset-svg/iron-iconset-svg.html
404 GET /components/todo-list/app/bower_components/paper-checkbox/paper-checkbox.html
404 GET /components/todo-list/app/bower_components/paper-material/paper-material.html
firefox 40               Tests passed
```

I simply hacked the existing my-greeting-basic.html like so 

```
<!doctype html>
<html>
<head>
  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
  <title>my-greeting-basic</title>

  <script src="../../bower_components/webcomponentsjs/webcomponents.min.js"></script>
  <link rel="import" href="../../bower_components/polymer/polymer.html">
  <script src="../../bower_components/web-component-tester/browser.js"></script>
  <script src="../../bower_components/test-fixture/test-fixture-mocha.js"></script>
  <link rel="import" href="../../bower_components/test-fixture/test-fixture.html">

  <!-- Step 1: import the element to test -->
  <link rel="import" href="../elements/todo-view/todo-view.html">

</head>
<body>

  <test-fixture id="basic">
    <template>
      <todo-view></todo-view>
    </template>
  </test-fixture>

  <script>

    suite('my-greeting tests', function() {
      var greeting, header;

      setup(function() {
        greeting = fixture('basic');
      });

      test('Welcome!', function() {
//        header = greeting.querySelector('h1');
        assert.equal('Welcome!', 'Welcome!');
      });

    });
  </script>

</body>
</html>
```

but am getting all those 404 errors. May I know why and how do i fix it? Thanks.

**Comment by johnlim** [comment]
While I did not root cause the issue, I suspect it to be a path issue related to where the files are served from (when running unit tests). I updated the build config files (gulpfile.js, .bowerrc etc) from the current polymer-starter-kit project and the unit tests run without issue now. Perhaps because the bower_components directory resides inside the app directory now which fixes the suspected path issue.
I could issue a pull request for this if you would like but perhaps you have some other direction intended for this project.
